packages/ERTP/src/issuerKit.js

```diff
@@ -61,3 +89,5 @@ const makeIssuerKit = (
 harden(makeIssuerKit);
 
 export { makeIssuerKit };
+
+/** @typedef {ReturnType<typeof makeIssuerKit>} IssuerKit */
```

Comment:
I'm fond of this idiom for internal use, but for an external API, it seems more important to spell out the type. Callers should not need to look at the implementation of `makeIssuerKit` to find it out this type. Yes, an IDE can figure it out for you, but we don't always look at code via an IDE.

somewhat related: generating documentation: #4291 . If we can use `tsc` to generate the spelled out type for documentation purposes, maybe this would be OK. But let's not presume that in this PR.